google/ggrc-core
# GGRC-3584 Several object roles are assigned to one person

# Issue description

The issue was that after import one person could be assigned several roles simultaneously. Which is incorrect. In case a new role is assigned to a person, the role with the higher privileges should be left and other roles deleted.

# Steps to test the changes

1. Create a program
2. Export created program as csv file
3. Open csv file 
4. Add to reader column a user that is displayed in Manager column > Save
5. Import csv file with program
6. Go to the program info page > People tab
7. Look at the tree view: Program manager is displayed as 'Program manager+1'
Expected Result: If user has several roles while importing, the highest role should be taken for the program

# Solution description

What was done in the PR is the following:
I pull all the user roles for the object being imported and check if they conflict. Mark the conflicting roles for deletion and delete them in column handler while inserting new objects into the db.

# Sanity checklist


# PR Review checklist
